ckanext/subscribe/action.py

```diff
@@ -38,13 +56,28 @@ def subscribe_signup(context, data_dict):
     :param skip_verification: Doesn't send email - instead it marks the
         subscription as verified. Can be used by sysadmins only.
         (optional, default=False)
+    :param g-recaptcha-response: Recaptcha response from frontend signup form
 
     :returns: the newly created subscription
     :rtype: dictionary
 
     '''
     model = context['model']
 
+    # Verify reCAPTCHA response
+    # Recaptcha response from backend signup form
+    # comes in as a request params
+    recaptcha_response = tk.request.params.get('g-recaptcha-response')
+    if not recaptcha_response:
+        # Recaptcha response from frontend signup form
```

Comment:
"frontend" and "backend" refer to our (ogdch) site setup and would be confusing to other users of this ckanext - can you change this please? Also in lines 59 and 68.